OMPython/__init__.py

```diff
@@ -698,6 +698,7 @@ def __init__(self, readonly=False, timeout = 10.00, docker = None, dockerContain
         # set omc executable path and args
         self._set_omc_command([
                                "--interactive=zmq",
+                               "--locale=C",
```

Comment:
This won't work with some older OMC versions

Reply:
Are you talking about like 1.12 or later. Anyway people using such old OMC can use older pip install version of OMPython.

Reply:
then that information concerning the minimum OMC version should probably be put somewhere (Readme, setup.py, warning at import time?)

Reply:
I load 1.9 versions of OMC that are older than this, and they work just fine... For zmq you could put it, but I guess not CORBA